server/test/com/xsn/explorer/data/BlockPostgresDataHandlerSpec.scala

```diff
@@ -234,7 +234,7 @@ class BlockPostgresDataHandlerSpec extends PostgresDataHandlerSpec with BeforeAn
         .copy(previousBlockhash = None, nextBlockhash = None)
       insert(block).isGood mustEqual true
 
-      val header = block.into[BlockHeader.Simple].transform
+      val header = block.into[BlockHeader.Simple].withFieldConst(_.transactions, 0).transform
```

Comment:
Missing tests for the new queries, it's crucial to test those, otherwise, sql syntax errors would get in production